EIPS/eip-5496.md

```diff
@@ -1,10 +1,10 @@
 ---
 eip: 5496
 title: Multi-privilege Management NFT Extension
-description: Create sharable multi-privilege NFTs for EIP-721
+description: Create shareable multi-privilege NFTs for EIP-721
 author: Jeremy Z (@wnft)
 discussions-to: https://ethereum-magicians.org/t/eip-5496-multi-privilege-management-extension-for-erc-721/10427
-status: Draft
+status: Last Call
 type: Standards Track
 category: ERC
 created: 2022-7-30
```

Comment:
This will probably be one of the most valuable expandable use case I could suggest: Consider mentioning how with EIP-1271 this EIP-5496 could be useful to authorize a privilege with smart contract, ideally providing an reference implementation. If you don't know how to do it, feel free to ask question in the [etheruem-magicians.org thread](https://ethereum-magicians.org/t/eip-5496-multi-privilege-management-extension-for-erc-721/10427) thread

Reply:
We will add new description which for Compliant contract, it is RECOMMENDED to use EIP-1271 to validate the signatures.